ts/features/wallet/store/reducers/bottomSheet.ts

```diff
@@ -0,0 +1,42 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { PersistConfig, persistReducer } from "redux-persist";
+import { getType } from "typesafe-actions";
+import { Action } from "../../../../store/actions/types";
+import { walletSetBottomSheetSurveyVisible } from "../actions/bottomSheet";
+
+export type WalletBottomSheetSurveyState = {
+  bottomSheetSurveyVisible: boolean;
+};
+
+const INITIAL_STATE: WalletBottomSheetSurveyState = {
+  bottomSheetSurveyVisible: true
```

Comment:
Can we move this to the IT-Wallet's `preferences` reducer? I think it will be a better fit, and we can avoid to create an entire reducer for this  specific purpose.
Also, can we add more context to the attribute name? `bottomSheetSurveyVisible` is too generic